qe_pre_installation_tasks.yml

```diff
@@ -20,7 +20,7 @@
       retries: 5
       delay: 5
       register: result
-      until: 'result|success'
+      until: 'result is success'
```

Comment:
@dahorak btw do you know why we have this this statement in quotes in some places, but without quotes in others?

Reply:
@mbukatov good question... and honest answer is, that I'm not sure... I think, that it should be safe to use it without quotes, but I'm not sure if there is any rule...